src/Model/GroundWaterFlow/gwf3maw8.f90

```diff
@@ -2506,36 +2521,29 @@ subroutine maw_fn(this, rhs, ia, idxglo, amatsln)
             end if
           end if
         end if
-        !
-        ! -- add maw storage changes
-        if (this%imawiss /= 1) then
-          if (this%ifwdischarge(n) /= 1) then
-            rate = this%area(n) * hmaw / delt
-            rterm = -rate
-            !
-            ! -- calculate storage derivative
-            drterm = -this%area(n) / delt
-            !
-            ! -- fill amat and rhs with storage components
-            rhs(iloc) = rhs(iloc) - rterm + drterm * hmaw
-          end if
-        end if
+        !!
```

Comment:
Should this code be commented out?

Reply:
Yes. Removed code since it was adding zero to the `RHS`.